### PR TITLE
Acceptance tests

### PR DESCRIPTION
**Pull request by varp**
Need a code review

### CODE REVIEW

**Comment by sosedoff** [comment]
There seems to be an issue with deps. Some of the packages are missing. Don't change the deps unless you mean to. That's why all the CI specs are failing (linux/win).

**Comment by sosedoff** [comment]
Script `script/test_ginkgo.sh` seems to be doing something odd. At first it tries to build a specific image, then goes on to test on various versions of postgres. The goal here is to utilize the `make test` and `make test-all` to run the full test suite.

File: script/test_ginkgo.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+set -e
+
+export PGWEB_VERSION=0.9.9
+export PGWEB_PORT=8081
+export PGHOST=${PGHOST:-localhost}
+export PGUSER="postgres"
+export PGPASSWORD=""
+export PGDATABASE="booktown"
+export PGPORT="15432"
+
```
Comment (varp) [inline script/test_ginkgo.sh]: @sosedoff First it starts pgweb's container for acceptance testing

File: script/test_ginkgo.sh [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+for i in {1..6}
+do
+  export PGVERSION="9.$i"
+
+  echo "---------------- BEGIN TEST ----------------"
+  echo "Running acceptance tests against PostgreSQL v$PGVERSION"
+
+  docker rm -f postgres || true
+  docker run -p $PGPORT:5432 --name postgres -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:$PGVERSION
```
Comment (varp) [inline script/test_ginkgo.sh]: There I'm starting different PG versions to which I will connect from PGWEB_CONTAINER in acceptance tests

File: script/test_ginkgo.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  sleep 5
+  ginkgo spec/...
+  echo "---------------- END TEST ------------------"
```
Comment (varp) [inline script/test_ginkgo.sh]: Here I'm running acceptance test from the PGWEB_CONTINAER to different PG backend

File: script/test_ginkgo.sh [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+do
+  export PGVERSION="9.$i"
+
+  echo "---------------- BEGIN TEST ----------------"
+  echo "Running acceptance tests against PostgreSQL v$PGVERSION"
+
+  docker rm -f postgres || true
+  docker run -p $PGPORT:5432 --name postgres -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:$PGVERSION
+  sleep 5
+  ginkgo spec/...
```
Comment (varp) [inline script/test_ginkgo.sh]: Here I'm running acceptance test from the PGWEB_CONTINAER to different PG backend

**Comment by varp** [comment]
About `script/test_ginkgo.sh`. I left comments in PR with explanations of goal which I'am trying to reach. Shortly, 
- run pgweb GUI from docker image
- looping through PG 9.1 - 9.6 backends and on each step we test pgweb GUI against different versions of Postgres

**Pull request closed without merging**